omnibus/config/software/oc_id.rb

```diff
@@ -21,6 +21,7 @@
 license "Apache-2.0"
 license_file "LICENSE"
 
+dependency "nokogiri" # for
```

Comment:
I see you tried to put a comment to explain why we are adding this. Is this part and intent to fix the build process?